src/ngraph/frontend/onnx_import/op/gemm.cpp

```diff
@@ -30,9 +31,19 @@ namespace ngraph
                 NodeVector gemm(const Node& node)
                 {
                     NodeVector inputs{node.get_ng_inputs()};
-                    auto input_a = inputs.at(0);
-                    auto input_b = inputs.at(1);
-                    auto input_c = inputs.at(2);
+                    std::shared_ptr<ngraph::Node> input_a = inputs.at(0);
+                    std::shared_ptr<ngraph::Node> input_b = inputs.at(1);
+                    std::shared_ptr<ngraph::Node> input_c;
+
+                    if (inputs.size() == 3)
+                    {
+                        input_c = inputs.at(2);
+                    }
+                    else
+                    {
+                        input_c = ngraph::op::Constant::create(
+                            input_b->get_element_type(), input_b->get_shape(), {0});
```

Comment:
You can make this constant as a scalar because this value should be broadcasted by auto-broadcast later.